src/components/VariantSearchField.tsx

```diff
@@ -343,7 +350,11 @@ export const VariantSearchField = ({ onVariantSelect, currentSelection, triggerS
       onDragLeave={dragLeave}
       onDrop={drop}
       className={
-        'p-1 m-1 border-2 border-dashed ' + (dragOngoingDepth ? 'border-black' : 'border-transparent')
+        `${
+          analysisMode === AnalysisMode.CompareToBaseline || analysisMode === AnalysisMode.CompareEquals
```

Comment:
I don't think that the `VariantSearchField` need/should know about the analysis mode. This component is about a single search field. It shouldn't care about for which analysis the selected variant is used and also not about how the field is positioned on the page. That should be the task of the component that uses it.

For the margins in this case, can't you set them instead in `VariantSearch`, e.g., around line 179?

![image](https://user-images.githubusercontent.com/18666552/185594467-fce45a87-152a-41e7-9b40-106df72d266d.png)